base_rest/restapi.py

```diff
@@ -161,10 +161,10 @@ def _binary_content_schema(self):
             for mediatype in self._mediatypes
         }
 
-    def to_openapi_requestbody(self, service):
+    def to_openapi_requestbody(self, services, spec):
```

Comment:
I have the feeling that `services` is an error here and should be singular `service`